### DISCUSSION TITLE
Appimage PATH Issues

### DISCUSSION THREAD

**Discussion by sonicnkt** [LizardByte/community#52] [Sunshine] [answered]
Im currently running 0.14 as Appimage on Manjaro, have been using AUR before but this is stuck at 0.13 atm. 
Anyway it works great and better than the old version but i am having an issue with custom environment variables that are set when you launch it:
```
Setting $HOME to /home/user/sunshine.AppImage.home
Setting $XDG_CONFIG_HOME to /home/user/sunshine.AppImage.config
```
As this sets custom home/config directories everything launched by sunshine will have issues, for example steam will setup a new install in `home/user/sunshine.AppImage.home/.steam` instead of starting you installed version. You can fix this tho by resetting those variables to the defaults in the launch command.

I could get everything working in the end but it is a bit annoying :) and i was wondering if its possible to configure the appimage to not touch these paths?

**Comment by ReenigneArcher** [comment]
It's not possible currently without refactoring the sunshine code base.

**Comment by ReenigneArcher** [comment, marked as answer]
This is solved in v0.15.0
